src/pages/Search/Search.js

```diff
@@ -56,10 +56,26 @@ function Search({ setData, setActiveCell }) {
 
 
   useEffect(() => {
+    function sortData(items) {
+      const sortPosts = new Array(7).fill([]).map(() => new Array(24).fill([]));
+      const timezoneDifference = () => new Date().getTimezoneOffset() * 60;
+
+      items.forEach((item) => {
+        const clientDate = (item.data.created_utc + timezoneDifference()) * 1000;
+        const date = new Date(clientDate);
+        const day = date.getDay();
+        const time = Number((`0${date.getHours()}`).slice(-2));
+        // eslint-disable-next-line
+        sortPosts[day][time] = sortPosts[day][time].concat(item.data);
+      });
+
+      setData(sortPosts);
```

Comment:
This can be a bit more descriptive. What about `setPosts` as action name? Also the state variable and so on can be called posts now I guess :smile: